scan/src/bindings/BandWeb3.re

```diff
@@ -42,7 +42,20 @@ let getAccounts = (instance, address) => {
   );
 };
 
+let addPublicKeyToSignedMsg: signed_msg_t => unit = [%bs.raw
+  {|
+function(signedMsg) {
+  for (const sig of signedMsg.tx.signatures) {
+    sig.public_key = Buffer.from(
+      'eb5ae98721' + Buffer.from(sig.pub_key.value, 'base64').toString('hex'), 'hex'
```

Comment:
`sha256("tendermint/PubKeySecp256k1")` = f8ccea**eb5ae987**ea423e6cc0e94297a53bd6862df3b3a02a6f6fc89250308760`

We use eb5ae987 AND 21 (= 21 base 16 = 33 = pubkey size)